src/sentry/models/grouphash.py

```diff
@@ -26,14 +26,10 @@ class State:
         UNLOCKED = None
         LOCKED_IN_MIGRATION = 1
 
-        # This hierarchical grouphash should be ignored/skipped for finding the group.
-        SPLIT = 2
-
     project = FlexibleForeignKey("sentry.Project", null=True)
     hash = models.CharField(max_length=32)
     group = FlexibleForeignKey("sentry.Group", null=True)
 
-    # not-null => the event should be discarded
```

Comment:
Why get rid of this comment? I think it's useful (and besides, it has nothing to do with hierarchical grouping).